sample-plugin/pom.xml

```diff
@@ -74,6 +74,16 @@
             <artifactId>checks-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>commons-lang3-api</artifactId>
```

Comment:
Not great (as mentioned, the idea is that transitive deps should be clean) but looks like the best we can do for now.